src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/card/card.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'cd-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss']
+})
+export class CardComponent {
+  @Input()
+  cardTitle: string;
+  @Input()
+  link: string;
```

Comment:
Probably we can prescind from this input arg for the time being (let's not overengineer).